### PR TITLE
RHPAM-2198 - NullPointerException in org.drools.core.marshalling.impl…

### PR DESCRIPTION
**Pull request by mswiderski**
….ProtobufOutputMarshaller.writeTimers in a multi-node setup

### CODE REVIEW

File: jbpm-services/jbpm-services-ejb/jbpm-services-ejb-timer/src/main/java/org/jbpm/services/ejb/timer/EJBTimerScheduler.java [hunk 1]
```diff
@@ -153,28 +154,31 @@ public TimerJobInstance getTimerByName(String jobName) {
...
+
     				if (handle.getUuid().equals(jobName)) {
-    					logger.debug("Job  {} does match timer and is going to be returned", jobName);
     					found = handle.getTimerJobInstance();
+							if (USE_LOCAL_CACHE) {
+    					    localCache.putIfAbsent(jobName, found);
+						  }
+    					logger.debug("Job {} does match timer and is going to be returned {}", jobName, found);
+
+    					break;
```
Comment (MarianMacik) [inline jbpm-services/jbpm-services-ejb/jbpm-services-ejb-timer/src/main/java/org/jbpm/services/ejb/timer/EJBTimerScheduler.java]: Formatting is quite wild here.

**Comment by tsurdilo** [comment]
Jenkins retest please

**Comment by mareknovotny** [comment]
what we will do with https://github.com/kiegroup/jbpm/pull/1509 ?

**Comment by MarianMacik** [comment]
@mareknovotny #1509 is for 7.18.x. But I think there will be no other patches, so we can probably close this.

**Comment by mareknovotny** [comment]
@MarianMacik exactly we won't do any patches for 7.3.x

**Pull request merged**